src/containerapp/azext_containerapp/custom.py

```diff
@@ -2637,3 +2641,240 @@ def list_environment_telemetry_otlp(cmd,
+    session_code_interpreter_decorater.register_provider(CONTAINER_APPS_RP)
+
+    session_code_interpreter_decorater.construct_payload()
+    r = session_code_interpreter_decorater.execute()
+
+    return r
+
+def upload_session_code_interpreter(cmd,
+                        name,
+                        resource_group_name,
+                        identifier, 
+                        filepath
+                        ):
+    raw_parameters = locals()
+    session_code_interpreter_decorater = SessionCodeInterpreterCommandsPreviewDecorator(
+        cmd=cmd,
+        client=SessionCodeInterpreterPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    session_code_interpreter_decorater.register_provider(CONTAINER_APPS_RP)
+    
+    r = session_code_interpreter_decorater.upload()
+
+    return r
+
+def show_file_content_session_code_interpreter(cmd,
+                        name,
+                        resource_group_name,
+                        identifier, 
+                        filename
+                        ):
+    raw_parameters = locals()
+    session_code_interpreter_decorater = SessionCodeInterpreterCommandsPreviewDecorator(
+        cmd=cmd,
+        client=SessionCodeInterpreterPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    session_code_interpreter_decorater.register_provider(CONTAINER_APPS_RP)
+
+    r = session_code_interpreter_decorater.show_file_content()
+
+    return r
+
+def show_file_metadata_session_code_interpreter(cmd,
+                        name,
+                        resource_group_name,
+                        identifier, 
+                        filename
+                        ):
+    raw_parameters = locals()
+    session_code_interpreter_decorater = SessionCodeInterpreterCommandsPreviewDecorator(
+        cmd=cmd,
+        client=SessionCodeInterpreterPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    session_code_interpreter_decorater.register_provider(CONTAINER_APPS_RP)
+
+    r = session_code_interpreter_decorater.show_file_metadata()
+
+    return r
+
+def list_files_session_code_interpreter(cmd,
+                        name,
+                        resource_group_name,
+                        identifier,
+                        path=None
+                        ):
+    raw_parameters = locals()
+    session_code_interpreter_decorater = SessionCodeInterpreterCommandsPreviewDecorator(
+        cmd=cmd,
+        client=SessionCodeInterpreterPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    session_code_interpreter_decorater.register_provider(CONTAINER_APPS_RP)
+
+    r = session_code_interpreter_decorater.list_files()
+
+    return r
+
+def delete_file_session_code_interpreter(cmd,
+                        name,
+                        resource_group_name,
```

Comment:
`resource_group_name,` is not required, same as above.

Reply:
same answer as above